packages/node/test/e2e/cli-util.js

```diff
@@ -20,45 +20,89 @@ export const CLI_PATH = fileURLToPath(
 
 /**
  * @import {ExecFileException} from 'node:child_process'
- * @import {RunCLIOutput} from '../types.js'
+ * @import {ExitCode, RunCLIOptions, RunCLIOutput} from '../types.js'
  * @import {ExecutionContext} from 'ava'
  */
 
 /**
  * Run the `@lavamoat/node` CLI with the provided arguments
  *
  * @template [Context=unknown] Default is `unknown`
+ * @overload
+ * @param {string[]} args CLI arguments
+ * @param {RunCLIOptions} [options] Options
+ * @returns {Promise<RunCLIOutput>}
+ */
+
+/**
+ * Run the `@lavamoat/node` CLI with the provided arguments
+ *
+ * @template [Context=unknown] Default is `unknown`
+ * @overload
  * @param {string[]} args CLI arguments
  * @param {ExecutionContext<Context>} [t] AVA test context
+ * @param {RunCLIOptions} [options] Options
  * @returns {Promise<RunCLIOutput>}
  */
 
-export const runCLI = async (args, t) => {
+/**
+ * Run the `@lavamoat/node` CLI with the provided arguments
+ *
+ * @template [Context=unknown] Default is `unknown`
+ * @param {string[]} args CLI arguments
+ * @param {ExecutionContext<Context> | RunCLIOptions} [tOrOptions] AVA test
```

Comment:
was expecting `options`, curious what the `tOr` prefix means?

Reply:
the two docstrings above this are also for this function; this is how you overload a function using jsdoc (`@overload` tag).  In this case, we have two overloads; one which expects the second parameter to be an AVA test context object; the other expects options (or undefined).  In the former, the second param is `t` and the third is `options`; in the latter, the second param is `options` and there is no third param.  Since, in the implementation, the second param will be `t` or `options`, it becomes `tOrOptions`. 😄